poll.js

```diff
@@ -62,6 +64,26 @@ class PollTxMonitor {
       statusUpdate.from = from;
       statusUpdate.to = to;
       statusUpdate.value = value;
+    } else if (type.includes('cosmos')) {
+      // TODO: handle cosmos transfer value if needed
+    } else if (receipt && (type === 'transfer' || type === 'transferDelegated')) {
+      // replace ETH LIKE transfer value
+      const transfers = getWeb3TransferFromReceipt(receipt);
+      let tx = transfers.find(
+        transfer => transfer.to.toLowerCase() !== networkTx.from.toLowerCase(),
+      );
+      if (!tx) {
+        const txs = transfers.filter(transfer => (new BigNumber(transfer.value)).gt(0));
+        if (txs.length) {
+          if (txs.length === 1) [tx] = txs;
+          else [tx] = txs.slice(-1); // assume last entry is tranfser
+        }
+      }
```

Comment:
If can accept transactions not sent by us, then should also confirm that `receipt.to === LIKE_COIN_ADDRESS`